examples/example-flutter/lib/ui/views/startup/startup_view.dart

```diff
@@ -15,7 +15,10 @@ class StartupView extends StackedView<StartupViewModel> {
   ) {
     return Scaffold(
       body: ExpressionView(
-        artboardName: 'intro-view',
+        artboardName:
+            MediaQuery.of(context).platformBrightness == Brightness.light
```

Comment:
This removes the Startup view intro. 

Can you please instead create a new view called themeView and add these changes in there. 

I'll link them all together in the next update of the example app.